### PR TITLE
Add the ostrich recipe

### PR DESCRIPTION
**Pull request by Zeitsperre**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Zeitsperre** [comment]
@julemai Can you confirm that you (also) wish to be a maintainer (on this as well)? Thanks!

File: recipes/ostrich/build.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Temporarily fix stuff with the GH ostrich repo.
+# (1) Replace "g++" by $(CXX) in the Makefile
+sed -i  -E 's/g\+\+/\$\(CXX\)/' make/makefile
+# v17.12.19 version: sed -i  -E 's/g\+\+/\$\(CXX\)/' makefile
+# (2) Add missing header
+sed -i  -E '/#include "MyHeaderInc.h"/a #include <iostream>' include/FileList.h
```
Comment (julemai) [inline recipes/ostrich/build.sh:6]: I don't think you need that fix anymore.

File: recipes/ostrich/meta.yaml [hunk 2]
```diff
@@ -29,7 +29,7 @@ about:
   doc_source_url: https://github.com/usbr/{{ name|lower }}/tree/master/docs
   license: GPL-2.0-only
   license_file: LICENSE
-  summary: This is an experimental conda binary for the Ostrich model calibration tool.
+  summary: This the conda-forge binary for the Ostrich hydrological model calibration tool.
```
Comment (julemai) [inline recipes/ostrich/meta.yaml:33]: Maybe remove "hydrological"?! Ostrich can be used for any kind of model

**Review by julemai — COMMENTED** [review]
Everything looks good to me- besides those two comments

**Comment by Zeitsperre** [comment]
@conda-forge/core This is good to be merged, thanks!

File: recipes/ostrich/build.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Temporarily fix stuff with the GH ostrich repo.
+# (1) Replace "g++" by $(CXX) in the Makefile
+sed -i  -E 's/g\+\+/\$\(CXX\)/' make/makefile
+
+# Compile
+cd make
+make GCC
```
Comment (ocefpaf) [inline recipes/ostrich/build.sh:7]: Is there a make check/test that we can run here?

Comment (Zeitsperre) [reply recipes/ostrich/build.sh:7]: No, unfortunately there isn't I can raise an issue for it upstream as that would be good practice. Not much I can do about it.

**Comment by Zeitsperre** [comment]
@lsmatott and @dloney can you indicate here that you would like to be maintainers? Thanks!

File: recipes/ostrich/build.sh (hunk 3 above)
Comment (Zeitsperre) [reply recipes/ostrich/build.sh:7]: The issue to eventually make tests within the package is up. Good for now?

**Comment by Zeitsperre** [comment]
@conda-forge/core Thanks again for the help, all users have indicated their interest in becoming package maintainers. Feel free to merge!

**Comment by ocefpaf** [comment]
> @conda-forge/core Thanks again for the help, all users have indicated their interest in becoming package maintainers. Feel free to merge!

Please ping the staged-recipes team instead of the core team for reviews here.

**Pull request merged**